File: sueldo.cpp
#include <iostream>
using namespace std;

string nombre;
float saldo;
int horas; 

int main(){
  cout << "Ingrese su nombre completo: " <<endl;
  cin >> nombre;
  cout << "Ingrese salario basico por hora: " <<endl;
  cin >>saldo;
  cout << "Ingrese las horas trabajadas en el mes: " <<endl;
  cin >>horas;
  
saldo = saldo * horas;
  
  if(saldo >= 450000){
     cout << "Su nombre es: " << nombre << endl;
     cout << "Su sueldo es: " << saldo << endl;
    
    }
  else{
    cout << "Su nombre es: " << nombre << endl;
  }
}

File: sueldo.py
nombre = str(input("Ingrese su nombre completo: "))
sueldo = float(input("Ingrese su sueldo por hora: "))
horas = int(input("Ingrese el numero de horas trabajadas en el mes: "))

sueldo = sueldo * horas

if sueldo >= 450000:
  print("Su nombre es:", nombre)
  print("Su sueldo es:", sueldo)
else: 
  print("Su nombre es:", nombre)